deriva/config/catalog_config.py

```diff
@@ -0,0 +1,137 @@
+"""DERIVA catalog configuration utilities."""
+
+import logging
+from deriva.core import tag, urlquote
+from deriva.core.ermrest_model import Schema
+
+logger = logging.getLogger(__name__)
+
+
+def configure_baseline_ermrest_client(model, apply=True):
```

Comment:
I'm also wondering whether these functions should actually be methods on the Model class...?

Reply:
Happy to move it there. So all three of these would become `Model.configure_baseline_...`.

Note: I am not aware of any wrappers/calls to these in the ermrest_model. In an earlier commit I actually started with them in ermrest_model and then moved them to their own module. Maybe you saw an earlier commit.

Reply:
done